lightning/src/routing/router.rs

```diff
@@ -1588,96 +1643,81 @@ where L::Target: Logger {
 		return Err(LightningError{err: "Failed to find a sufficient route to the given destination".to_owned(), action: ErrorAction::IgnoreError});
 	}
 
-	// Sort by total fees and take the best paths.
-	payment_paths.sort_unstable_by_key(|path| path.get_total_fee_paid_msat());
-	if payment_paths.len() > 50 {
-		payment_paths.truncate(50);
-	}
-
-	// Draw multiple sufficient routes by randomly combining the selected paths.
-	let mut drawn_routes = Vec::new();
-	let mut prng = ChaCha20::new(random_seed_bytes, &[0u8; 12]);
-	let mut random_index_bytes = [0u8; ::core::mem::size_of::<usize>()];
-
-	let num_permutations = payment_paths.len();
-	for _ in 0..num_permutations {
-		let mut cur_route = Vec::<PaymentPath>::new();
-		let mut aggregate_route_value_msat = 0;
-
-		// Step (6).
-		// Do a Fisher-Yates shuffle to create a random permutation of the payment paths
-		for cur_index in (1..payment_paths.len()).rev() {
-			prng.process_in_place(&mut random_index_bytes);
-			let random_index = usize::from_be_bytes(random_index_bytes).wrapping_rem(cur_index+1);
-			payment_paths.swap(cur_index, random_index);
+	// Step (6).
+	let mut selected_route = payment_paths;
```

Comment:
Since this is a simple renaming/move, I assume it should be a no-op after the compiler is done with it?